core/chains/evm/orm.go

```diff
@@ -0,0 +1,68 @@
+package evm
+
+import (
+	"github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/smartcontractkit/sqlx"
+	null "gopkg.in/guregu/null.v4"
+)
+
+type ORM interface {
+	CreateChain(id utils.Big, config types.ChainCfg) (types.Chain, error)
+	DeleteChain(id utils.Big) error
+	Chains(offset, limit int) ([]types.Chain, int, error)
+	CreateNode(data NewNode) (types.Node, error)
+	DeleteNode(id int64) error
+	Nodes(offset, limit int) ([]types.Node, int, error)
+}
+
+type orm struct {
+	db *sqlx.DB
+}
+
+var _ ORM = (*orm)(nil)
+
+func NewORM(db *sqlx.DB) ORM {
+	return &orm{db}
+}
+func (o *orm) CreateChain(id utils.Big, config types.ChainCfg) (chain types.Chain, err error) {
+	sql := `INSERT INTO evm_chains (id, cfg, created_at, updated_at) VALUES ($1, $2, now(), now()) RETURNING *`
```

Comment:
@samsondav If we set the column's default value to `now()` then we wouldn't need to do this explicitly in the insert, WDYT?

Reply:
I didn't know you could do that.

I'm torn because gorm autofills these with `time.Now()`. Application time may be different than DB time so probably it makes sense to use `time.Now` instead for consistency (although I also get lazy and don't always use it).